modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
 
 == Use Case: Studio
 
-In the following example, a Mule application sends a message to a phone number, and then redacts it.
+In the following example, a Mule application sends a message to a phone number, and then redacts it. This example uses variables for field values. You can provide the values for each variable in a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
 
-. Create a new Mule application and add the following properties to the mule-app.properties file:
+. Create a new Mule application and add the following properties to the properties file:
```

Comment:
I'm confused: the example also uses a properties file?
Maybe we don't need the long explanation above, if the example already assumes that the audience knows how to use a properties file. Also, aren't most audiences savvy enough to recognize variable values?

Reply:
Part of the bug was that we should be explaining where the properties file is located, so I am leaving this in.